README.md

```diff
@@ -78,5 +72,37 @@ Now that the datasets are working, we only need to include them into CosmoScout
 },
 ...
 ```
-
 You should also remove the "Earth" section from the "csp-simple-bodies" plugin configuration in the same file, else you will have two Earths drawn on top of each other!
+
+## 2) Loading Custom Map Data
+Here you can run a dockerized mapserver using your own custom dataset.
+
+### Preparing the Map Data
+As you are using your own custom dataset some changes need to be made to the `meta.map` file in the `mapserver-datasets` directory. The lines `42,43 and 44` need to change accordingly to the location of your `.map` files.
```

Comment:
Please describe in more detail how to set up the data directory. Imagine you have a tiff file for some location on Earth, which steps are necessary to serve this with the mapserver? This definitely involves:
* Setting up a mapserver-datasets directory with an appropriate directory structure inside.
* Copying the epsg and meta.map files from the repository
*  Adding a custom map file for the tif
* Maybe optimizing the tif with `gdal_translate` and `gdaladdo`
* Running the container
* Finally configuring CosmoScout VR